saleor/static/dashboard-next/products/queries.ts

```diff
@@ -115,7 +115,7 @@ export const productFragmentDetails = gql`
         slug
       }
     }
-    availability {
+    pricing {
```

Comment:
Below we also use `pricing.available` while we should use `product.isAvailable`.